packages/@aws-cdk/toolkit/lib/types.ts

```diff
@@ -0,0 +1,78 @@
+export type ToolkitAction =
+  | 'bootstrap'
+  | 'synth'
+  | 'list'
+  | 'deploy'
+  | 'destroy';
+
+export type MessageLevel = 'error' | 'warn' | 'info' | 'debug';
+
+export enum StackSelectionStrategy {
+  /**
+   * Returns an empty selection in case there are no stacks.
+   */
+  None = 'none',
+
+  /**
+   * If the app includes a single stack, returns it. Otherwise throws an exception.
+   * This behavior is used by "deploy".
+   */
+  OnlySingle = 'single',
```

Comment:
I'm more familiar with enums being fully capitalized, like `ONLY_SINGLE`. curious why the enums in this file follow a different naming scheme